HelpSource/Classes/Pdiff.schelp

```diff
@@ -0,0 +1,17 @@
+class:: Pdiff
+summary:: returns the difference between the current and previous values of an enclosed pattern
+related:: Classes/Padd
```

Comment:
Padd is not really related (different purpose).

The inverse operation of differentiation is integration. It isn't obvious at first, but in pattern-land, the integrator is Pseries. So I'd put this one in as the related class.

Reply:
Thanks!  That makes a lot of sense, I have changed it.  I also added a file for Pdrop.